blocks/api/utils.js

```diff
@@ -72,7 +72,20 @@ export function normalizeIconObject( icon ) {
 
 	if ( icon.background ) {
 		const tinyBgColor = tinycolor( icon.background );
-		if ( ! icon.foreground ) {
+		let shouldComputeForeground = true;
+		if ( icon.foreground ) {
+			const tinyFgColor = tinycolor( icon.foreground );
```

Comment:
Can we use `foregroundColor` or just `foreground` instead of `FgColor` here? Abbreviations like this aren't very helpful and it's not like there aren't longer lines 😉